packages/runtime/src/errors/errors.ts

```diff
@@ -39,6 +41,44 @@ export class TealError extends Error {
   }
 }
 
+export class BuilderError extends Error {
```

Comment:
should we keep this in a separate file? Maybe `builder-errors.ts`. And rename this to `teal-errors.ts`?

Reply:
i am thinking of creating `runtime-error.ts` and restructrue erros properly in different task,, will update this one

Reply:
ok